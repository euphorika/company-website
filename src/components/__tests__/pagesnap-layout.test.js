import React from "react"
import { shallow } from "enzyme"
import toJson from 'enzyme-to-json'
import { useStaticQuery } from "gatsby"

import PageSnapLayout from "../pagesnap-layout"

beforeEach(() => {
  useStaticQuery.mockReturnValueOnce({
    site: {
      siteMetadata: {
        title: `Default Starter`
      }
    }
  })
})

describe("PageSnapLayout", () => {
  it("renders correctly", () => {
    const PageSnapLayoutElement = (
      <PageSnapLayout seoTitle="SEO Title">
        <div>Children</div>
      </PageSnapLayout>
    )

    const wrapper = shallow(PageSnapLayoutElement)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
