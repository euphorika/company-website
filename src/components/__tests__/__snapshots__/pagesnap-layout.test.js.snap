// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSnapLayout renders correctly 1`] = `
<Fragment>
  <SEO
    description=""
    keywords={Array []}
    lang="en"
    meta={Array []}
    title="SEO Title"
  />
  <Header
    siteTitle="Default Starter"
  />
  <PageSnapContainer>
    <div>
      Children
    </div>
    <FullPage
      backgroundColor="#00b8d2"
      headerFontColor="#ffffff"
    >
      <FullpageFooter />
    </FullPage>
  </PageSnapContainer>
</Fragment>
`;
